css/style.css

```
@@ -0,0 +1,109 @@
+/***************************/
+/* HEADER */
+/**************************/
+.header {
+  background-image: url(/img/Bg.png);
```

Checkout [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/background#syntax) to make sure you specify properties as it should.

Also it's better not to mess with capital letters in the file naming to keep it simple. It'd suggest to use kebab case. Also please give meaningful names. There could be a lot of different `bg`s in your project and you have to somehow distinguish them.
```suggestion
  background-image: url("/img/bg-promo-section.png");
```